partridge/writers.py

```diff
@@ -35,15 +36,22 @@ def write_feed_dangerously(feed, outpath, nodes=None):
     your own risk.
     """
     nodes = DEFAULT_NODES if nodes is None else nodes
+
     try:
```

Comment:
Would it be helpful to catch the exception within try and pass it back/otherwise inform the end use of what happened?

Reply:
Unless there's a specific error that's recoverable or can be interpreted better here than by caller, I would think it's better to let it propagate.

One approach, though, could be to fall back to trying to do it serially without the pool.

Reply:
Although, I'm not sure what would cause the thread pool to fail other than some problem with the individual work items, so maybe a fallback isn't helpful.